### PR TITLE
Update requirements of the app

### LINKED ISSUE #4: requirements for tags microservice
We need specify the correct requirements for run the tag service

### PR DESCRIPTION
**Pull request by cayop**
Adding minimum new requirements for up the tag service
[1643](https://tree.taiga.io/project/devicetag/task/1643)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -12,3 +12,9 @@ Flask-SQLAlchemy==2.3.2
 colorama==0.3.9
 colour==0.1.5
 psycopg2-binary==2.7.5
+webargs==4.0.0
+Werkzeug==0.14.1
+pytest==3.7.2
+pytest-runner==4.2
```
Comment (slamora) [inline requirements.txt]: Why include test related requirements as base requirements? Besides that I think that there is no tests on tag project.

File: requirements.txt [hunk 2]
```diff
@@ -12,3 +12,9 @@ Flask-SQLAlchemy==2.3.2
...
+alembic==1.4.2
```
Comment (slamora) [inline requirements.txt]: Alembic is used to manage migrations on devicehub-teal. Are you sure that is a requirement here?

File: requirements.txt [hunk 3]
```diff
@@ -12,3 +12,9 @@ Flask-SQLAlchemy==2.3.2
 colorama==0.3.9
 colour==0.1.5
 psycopg2-binary==2.7.5
+webargs==4.0.0
+Werkzeug==0.14.1
```
Comment (slamora) [inline requirements.txt]: AFAIK Werkzeug is a requirement of flask. I don't think that we need to add here.

**Review by slamora — CHANGES_REQUESTED** [review]
The issue #4 is related to check requirements versions or missing requirements. Please check it but keep as clean as possible requirements.txt dropping away unused libraries.

File: requirements.txt (hunk 2 above)
Comment (cayop) [reply requirements.txt]: The service tag have models and the view save tags in the database.
The database is other of the database of dh. 
I suppose that in the future maybe we need to do a migration of this. And yes I think is a good idea have Alembic for this task

File: requirements.txt (hunk 1 above)
Comment (cayop) [reply requirements.txt]: Now there are test and all is broken but I suppose that we want to do a tests

File: requirements.txt (hunk 3 above)
Comment (cayop) [reply requirements.txt]: I have this error without Werkzeug:
```
Traceback (most recent call last):
  File "examples/dh.py", line 4, in <module>
    from teal.teal import Teal
  File "/home/ribaguifi/ereuse-tag/tag/env/lib/python3.7/site-packages/teal/teal.py", line 15, in <module>
    from werkzeug.wsgi import DispatcherMiddleware
ImportError: cannot import name 'DispatcherMiddleware' from 'werkzeug.wsgi' (/home/ribaguifi/ereuse-tag/tag/env/lib/python3.7/site-packages/werkzeug/wsgi.py)
```

File: requirements.txt (hunk 1 above)
Comment (slamora) [reply requirements.txt]: What I mean is that requirements.txt is the file used by a user to deploy tag service. A normal user (not a developer) doesn't want to install dependencies only related to run tests.

So, my proposal is to move test related dependencies to a new file (e.g. requirements-test.txt).

File: requirements.txt (hunk 3 above)
Comment (slamora) [reply requirements.txt]: Ok! My fault, seems that Werkzeug is the dependency problem to fix #4.

File: requirements.txt (hunk 2 above)
Comment (slamora) [reply requirements.txt]: Yes, I agree with you. But we don't have any related code to alembic yet, so it's not a current requirement.

Comment (slamora) [reply requirements.txt]: Remember, we want to keep our code and our project as clean and tiny as posible.

File: requirements.txt (hunk 3 above)
Comment (cayop) [reply requirements.txt]: Ok, Now is done.
test requirements is in requirements-test.txt and alembic is out.

**Review by slamora — APPROVED** [review]

**Pull request merged**